lib/husky_musher/app.py

```diff
@@ -127,21 +135,41 @@ def main():
         # If not in REDCap project, create new record
```

Comment:
Its not directly related to adding attestations, but in the interest of making sure it's removed before launch, should we take out the testing netid block starting at 118 above? (Can't comment directly on the line because its to far from these actual changes 😅)

Reply:
Good catch! I've removed the test case 😅